source/tutorials.txt

```diff
@@ -13,18 +13,18 @@ Tutorials
 
 .. toctree::
 
-   /tutorials/connect/
-   /tutorials/db-coll/
-   /tutorials/indexes/
-   /tutorials/read-ops/
-   /tutorials/encrypt/
-   /tutorials/write-ops/
-   /tutorials/aggregation/
-   /tutorials/change-stream/
-   /tutorials/text-search/
-   /tutorials/geospatial/
-   /tutorials/gridfs/
-   /tutorials/command/
+   Connect to MongoDB </tutorials/connect/>
+   Databases & Collections </tutorials/db-coll/>
+   Indexes </tutorials/indexes/>
+   Read Operations </tutorials/read-ops/>
+   In-Use Encryption </tutorials/encrypt/>
+   Write Operations </tutorials/write-ops/>
+   Aggregation Framework </tutorials/aggregation/>
+   Change Streams </tutorials/change-stream/>
+   Text Search </tutorials/text-search/>
+   Geospatial Search </tutorials/geospatial/>
+   GridFS </tutorials/gridfs/>
+   Run Commands </tutorials/command/>
```

Comment:
S: I think another TOC said "Run Database Commands", which is a little more precise